README.md

```diff
@@ -118,56 +118,56 @@ include:
 </ros2_control>
 ```
 
-## Add the ign_ros2_control plugin
+## Add the gz_ros2_control plugin
 
 In addition to the `ros2_control` tags, a Gazebo plugin needs to be added to your URDF that
 actually parses the `ros2_control` tags and loads the appropriate hardware interfaces and
-controller manager. By default the `ign_ros2_control` plugin is very simple, though it is also
+controller manager. By default the `gz_ros2_control` plugin is very simple, though it is also
 extensible via an additional plugin architecture to allow power users to create their own custom
 robot hardware interfaces between `ros2_control` and Gazebo.
 
 ```xml
 <gazebo>
-    <plugin filename="libign_ros2_control-system.so" name="ign_ros2_control">
+    <plugin filename="libgz_ros2_control-system.so" name="gz_ros2_control">
```

Comment:
Nit: the `lib` and `.so` are optional, omitting them is better for cross-platform

```suggestion
    <plugin filename="gz_ros2_control-system" name="gz_ros2_control">
```